packages/flutter/lib/src/material/material_state_mixin.dart

```diff
@@ -0,0 +1,130 @@
+// Copyright 2021 The Flutter Authors. All rights reserved.
```

Comment:
I believe this has to be 2014. Also perhaps add an empty line after the license header